documentation/asciidoc/accessories/camera/synchronous_cameras.adoc

```diff
@@ -1,102 +1,97 @@
 == Synchronous Captures
 
-Both the HQ Camera and the Global Shutter Camera, have support for synchronous captures.
-Making use of the XVS pin (Vertical Sync) allows one camera to pulse when a frame capture is initiated.
-The other camera can then listen for this sync pulse, and capture a frame at the same time as the other camera.
+The High Quality (HQ) Camera supports synchronous captures.
+Using the XVS pin (Vertical Sync) allows one camera (the "source") to pulse when a frame capture is initiated.
```

Comment:
"to pulse" -> "to send a pulse" ?

Reply:
Yes. And I probably should re-check the datasheet and whether or not we're using "fast trigger mode" or whatever it is... to check that the final part of the sentence is accurate too.